### PR TITLE
fix(frontend): add "red" flag to OptionButton; make delete table button visible

### LINKED ISSUE #1905: 🐛 [Bug] Delete button not visible in my tables section
## 🐛 Bug
It looks like this:
Screenshot 2024-09-01 at 23 13 43
Should look like this:
Screenshot 2024-09-01 at 23 13 47

It's a CSS priority problem. The fix should be simple, but is not straightforward if we want to avoid `!important`.

### PR DESCRIPTION
**Pull request by Bettelstab**
## 🍰 Pullrequest
Makes the delete button in My Tables visible.
Gives `OptionButton` a flag `red`.

### Issues
- fixes #1905

### CODE REVIEW

File: frontend/src/components/cockpit/my-tables/TableItem.vue [hunk 1]
```diff
@@ -84,7 +84,7 @@ const deleteTable = () => {
 }
 </script>
 
-<style scoped lang="scss">
+<style scoped lang="css">
```
Comment (roschaefer) [inline frontend/src/components/cockpit/my-tables/TableItem.vue:87]:
```suggestion
<style scoped>
```

File: frontend/src/components/cockpit/my-tables/TableItem.vue [hunk 2]
```diff
@@ -138,11 +138,6 @@ const deleteTable = () => {
   }
 }
 
-.trash {
```
Comment (roschaefer) [inline frontend/src/components/cockpit/my-tables/TableItem.vue:141]: The less custom CSS, the better.

File: frontend/src/components/cockpit/options-list/OptionButton.vue [hunk 3]
```diff
@@ -31,6 +34,11 @@ defineEmits(['click'])
   }
 }
 
+.item.red {
```
Comment (roschaefer) [inline frontend/src/components/cockpit/options-list/OptionButton.vue:37]: Ah, well.

Comment (roschaefer) [reply frontend/src/components/cockpit/options-list/OptionButton.vue:37]: The stronger specificity of `.item.red` does the trick, turns the background red and makes the font color visible?

I would love to use Tailwind to never have to fight that war ever again.

**Review by roschaefer — DISMISSED** [review]
I will approve this but it's yet another issue with Vuetify.

File: frontend/src/components/cockpit/options-list/OptionButton.vue (hunk 3 above)
Comment (Bettelstab) [reply frontend/src/components/cockpit/options-list/OptionButton.vue:37]: Actually originally I preferred to just set the color from the parent component. That would not work with Tailwind either, I fear. With CSS modules there could be a way... but as we will use this color flag more times, it makes sense to have it like this.
Can't see whats wrong with this code now. It's super clear and does not feel like war, but like CSS.

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**